### DISCUSSION TITLE
What's the delay for app updates?

### DISCUSSION THREAD

**Discussion by borekb** [notaapp/talk#255] [Questions] [answered]
Version 0.37 is out for about 24 hours now (see e.g. https://github.com/notaapp/talk/discussions/250) but I didn't get it on any of my computers yet. I understand there is some delay and that it's probably intentional but could you share more about how it works and how much longer should I expect to wait?

(I know I could probably download a .dmg from `/releases` if I really wanted.)

**Comment by astoilkov** [comment, marked as answer]
You are right, it's a slow rollout. Every 15 minutes the app check for updates, each time you have a 1% chance to get the update. We disable that logic after a few days when we are sure the release doesn't have serious bugs.

0.37 includes a "Check for updates" command that when executed will overwrite that behavior. For the next releases, you could just execute the command and immediately get the release.

**Reply by borekb** [reply]
I'm definitely looking forward to the "Check for updates" command 😄.

Out of curiosity, do you use some 3rd party service for this gradual rollout?

**Reply by astoilkov** [reply]
No, it's a custom solution — a minimalistic solution that works great.

**Reply by borekb** [reply]
As everything you do 😄. Thanks.
